tcn_hpl/callbacks/plot_metrics.py

```diff
@@ -362,6 +364,40 @@ def on_test_epoch_end(
         test_acc = pl_module.test_acc.compute()
         test_f1 = pl_module.test_f1.compute()
 
+        # Create activity predictions KWCOCO JSON
+        truth_dset_fpath = trainer.datamodule.hparams["coco_test_activities"]
+        truth_dset = kwcoco.CocoDataset(truth_dset_fpath)
+        acts_dset = kwcoco.CocoDataset()
+        acts_dset.fpath = self._preds_dset_output_fpath
+        acts_dset.dataset['videos'] = truth_dset.dataset['videos']
+        acts_dset.dataset['images'] = truth_dset.dataset['images']
+        acts_dset.index.build(acts_dset)
+        acts_dset.dataset['categories'] = truth_dset.dataset['categories']
+        # Create numpy lookup tables
+        for i in range(len(all_preds)):
+            frame_index = all_source_frames[i].item()
+            video_id = all_source_vids[i].item()
+            # Now get the image_id that matches the frame_index and video_id.
+            sorted_img_ids_for_one_video = acts_dset.index.vidid_to_gids[int(video_id)]
+            image_id = sorted_img_ids_for_one_video[frame_index]
+            # Sanity check: this image_id corresponds to the frame_index and video_id
+            try:
+                assert acts_dset.index.imgs[image_id]['frame_index'] == frame_index
+                assert acts_dset.index.imgs[image_id]['video_id'] == video_id
+            except:
+                import ipdb; ipdb.set_trace()
```

Comment:
Take out the debugging
```suggestion
            assert acts_dset.index.imgs[image_id]['frame_index'] == frame_index
            assert acts_dset.index.imgs[image_id]['video_id'] == video_id
```